pkg/skaffold/jib/jib.go

```diff
@@ -229,3 +231,15 @@ func relativize(path string, roots ...string) (string, error) {
 	}
 	return "", errors.New("could not relativize path")
 }
+
+// isOnInsecureRegistry checks if the given image specifies an insecure registry
+func isOnInsecureRegistry(image string, insecureRegistries map[string]bool) bool {
+	ref, err := name.ParseReference(image)
+	if err != nil {
+		// ignore the error as the image should have been validated before this
+		return false
+	}
+
+	registry := ref.Context().Registry.Name()
```

Comment:
Just wondering how this map works in Skaffold. Will aliases work? Like for `openjdk:8`.

Reply:
It seems go-containerregistry's rewrites [an empty registry](https://github.com/google/go-containerregistry/blob/31e00cede111067bae48bfc2cbfc522b0b36207f/pkg/name/registry.go#L46) and [`docker.io`](https://github.com/google/go-containerregistry/blob/31e00cede111067bae48bfc2cbfc522b0b36207f/pkg/name/registry.go#L127-L131) to `index.docker.io` internally.